data-raw/standard-format-data-prep/covariate_stock_recruit.Rmd

```diff
@@ -53,10 +88,45 @@ ggplot(butte_dd_sum, aes(wyear, totalddays)) +
 ```
 
 
-## Day of year
+## Day of year when 7DADM drops below 13 C
+#Butte Creek
+```{r}
+#Filtering Butte data 
+butte_std_temp <- read.csv(here::here("data-raw", "standard-format-data-prep", "temp_data", "standard_temperature.csv")) |>
+  filter(stream == "butte creek")
+
+#Changing "date" field into date format
+butte_std_temp$date <- as.Date(butte_std_temp$date)
```

Comment:
This definitely works but lets move towards use tidyverse. You can add this line to the code above like this (note i usually start new line after each pipe but may not show up like that here):
`butte_std_temp <- strd_temp |> 
filter(stream == "butte creek" |> 
mutate(date = as.Date(date))`